test/ode/ode_cache_tests.jl

```diff
@@ -4,6 +4,7 @@ CACHE_TEST_ALGS = [Euler(),Midpoint(),RK4(),SSPRK22(),SSPRK33(),SSPRK53(), SSPRK
   SSPRK63(),SSPRK73(),SSPRK83(),SSPRK432(),SSPRK932(),SSPRK54(),SSPRK104(),
   ORK256(), CarpenterKennedy2N54(), HSLDDRK64(), DGLDDRK73_C(), DGLDDRK84_C(), DGLDDRK84_F(), NDBLSRK124(), NDBLSRK134(), NDBLSRK144(),
   CFRLDDRK64(), TSLDDRK74(),
+  CKLLDDRK43_2(),
```

Comment:
` @test_nowarn sol = solve(prob,alg,callback=callback,dt=1/2)`
This is reporting `ERROR: BoundsError` for `CKLLDDRK43_2`.

Reply:
I think there is some problem with the cache, some variables are not `resize`d correctly. But @ChrisRackauckas will definitively know more about that.

Reply:
Yeah, this is for testing how resize works, so you'd have to build based on the macro assumptions. I'll see I can spot the failure cause without digging in.